### PR TITLE
Adding sample client for is-analytics

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: modules/samples/sample-clients/is-analytics-client/build.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<!--
+  ~ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (Niveathika92) [inline modules/samples/sample-clients/is-analytics-client/build.xml]: License Year

File: modules/samples/sample-clients/is-analytics-client/pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
```
Comment (Niveathika92) [inline modules/samples/sample-clients/is-analytics-client/pom.xml]: Same

File: modules/samples/sample-clients/is-analytics-client/src/main/java/org/wso2/carbon/sample/isanalyticsclient/Client.java [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+import org.wso2.carbon.databridge.agent.DataPublisher;
+import org.wso2.carbon.databridge.commons.Event;
+import org.wso2.carbon.databridge.commons.utils.DataBridgeCommonsUtils;
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+/**
+ * WSO2Event Client Publisher.
```
Comment (Niveathika92) [inline modules/samples/sample-clients/is-analytics-client/src/main/java/org/wso2/carbon/sample/isanalyticsclient/Client.java]: Update Comment

File: modules/samples/sample-clients/is-analytics-client/pom.xml (hunk 2 above)
Comment (ThisaruGuruge) [reply modules/samples/sample-clients/is-analytics-client/pom.xml]: Updated

File: modules/samples/sample-clients/is-analytics-client/pom.xml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.sp</groupId>
+        <artifactId>sample-clients</artifactId>
+        <version>4.3.1-SNAPSHOT</version>
```
Comment (dilini-muthumala) [inline modules/samples/sample-clients/is-analytics-client/pom.xml]: Pls update this to 4.3.0-SNAPSHOT.

File: modules/samples/sample-clients/is-analytics-client/src/main/java/org/wso2/carbon/sample/isanalyticsclient/DataPublisherUtil.java [hunk 5]
```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (dilini-muthumala) [inline modules/samples/sample-clients/is-analytics-client/src/main/java/org/wso2/carbon/sample/isanalyticsclient/DataPublisherUtil.java]: Please fix year.

File: modules/samples/sample-clients/is-analytics-client/src/main/resources/data.agent.config.yaml [hunk 6]
```diff
@@ -0,0 +1,131 @@
+################################################################################
+#   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```
Comment (dilini-muthumala) [inline modules/samples/sample-clients/is-analytics-client/src/main/resources/data.agent.config.yaml]: Please fix year.

File: modules/samples/sample-clients/is-analytics-client/src/main/resources/log4j.properties [hunk 7]
```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (dilini-muthumala) [inline modules/samples/sample-clients/is-analytics-client/src/main/resources/log4j.properties]: please fix year.

File: modules/samples/sample-clients/is-analytics-client/src/main/resources/sync.data.agent.config.yaml [hunk 8]
```diff
@@ -0,0 +1,131 @@
+################################################################################
+#   Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved
```
Comment (dilini-muthumala) [inline modules/samples/sample-clients/is-analytics-client/src/main/resources/sync.data.agent.config.yaml]: please fix year.

**Pull request merged**